docs/docs/overview/performance.md

```diff
@@ -22,13 +22,15 @@ We used the following configurations for the benchmarks.
 
 - Nodes: 3
 - Machines: `n2d-standard-2`
+- CVM: `true`
 - Kubernetes version: `1.23.6-gke.2200`
 - Zone: `europe-west3-b`
 
 #### Constellation and AKS on Azure
 
 - Nodes: 3
-- Machines: `D2a_v4`
+- Machines: `DC4as_v5`
+- CVM: `true`
 - Kubernetes version: `1.23.5`
 - Region: `North Europe`
 - Zone: `2`
```

Comment:
Not sure if you pinned the zones for the benchmark, but AFAIK Constellation on Azure can scale to multiple zones by default (there’s no pinned zone), so this information could be slightly wrong for Constellation.

Reply:
No, I did not pin it. In fact I used `westeurope` (need to update the quoted section above) and no pinned zone. 

But this brings up another point: IIRC Azure does not support `DC4as_v5` or maybe any CVMs in Zone 3 of `westeurope`, don't they? So for this run, probably doesn't matter. But for future benchmarks, I'd definitely pin it. Thanks.

Reply:
Okay, if Azure only has VMs of this type in one zone in this region then I suppose you're lucky and that should be fine then for now :)